.github/workflows/run_rc_validation_go_wordcount.yml

```diff
@@ -83,9 +87,7 @@ jobs:
           go run wordcount.go \
             --input ./kinglear.txt \
             --output ./output_prism.txt \
-            --runner=PrismRunner \
-            --environment_type=DOCKER \
-            --environment_config=apache/beam_go_sdk:${{ github.event.inputs.container_tag }}
+            --runner=PrismRunner
```

Comment:
As this is currently written, it will not do anything with the rc version of the sdk - we're not downloading beam anywhere.

Reply:
If you look at https://github.com/apache/beam/actions/runs/14346450686/job/40217141111 you can see it is using 2.65.0-dev